complex.c

```diff
@@ -2506,6 +2506,79 @@ float_arg(VALUE self)
  * - Separately, with methods Complex#real and Complex#imag.
  * - Together, with method Complex#rect.
  *
+ * == What's Here
+ *
+ * First, what's elsewhere:
+ *
+ * - \Class \Complex inherits (directly or indirectly)
+ *   from classes {Numeric}[rdoc-ref:Numeric@What-27s+Here]
+ *   and {Object}[rdoc-ref:Object@What-27s+Here].
+ * - Includes (indirectly) module {Comparable}[rdoc-ref:Comparable@What-27s+Here].
+ *
+ * Here, class \Complex has methods for:
+ *
+ * - Creating \Complex objects:
```

Comment:
I noticed that other docs (e.g. [Array](https://docs.ruby-lang.org/en/master/Array.html#class-Array-label-What-27s+Here), [Hash](https://docs.ruby-lang.org/en/master/Hash.html#class-Hash-label-What-27s+Here), [Range](https://docs.ruby-lang.org/en/master/Range.html#class-Range-label-What-27s+Here)) use subtitles instead of bullet points. Should we do that here too for consistency?

Reply:
Done.